src/Illuminate/Database/DetectsLostConnections.php

```diff
@@ -63,6 +63,8 @@ protected function causedByLostConnection(Throwable $e)
             'Reason: Server is in script upgrade mode. Only administrator can connect at this time.',
             'Unknown $curl_error_code: 77',
             'SSL: Handshake timed out',
+            'SQLSTATE[08006] [7] SSL error: sslv3 alert unexpected message',
```

Comment:
sslv3 😭 do we really want to support use of SSL v3? Do any modern databases actually still support this?

Reply:
> AWS Aurora to using AWS RDS Proxy

Seems like a configuration error perhaps? Can you try setting the minimum TLS version to 1.2 in the parameter group?